CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 
 Here are a few things you can do that will increase the likelihood of your pull request being accepted:
 
-- Follow the [style guide][style] which is a custom eslint linter. Any linting errors should be shown when running `npm test`
+- Follow the [style guide](https://github.com/standard/eslint-config-standard) which is using standard. Any linting errors should be shown when running `npm test`
```

Comment:
Could you update this back to use `style` and then update the link here https://github.com/probot/probot/pull/272/files#diff-6a3371457528722a734f3c51d9238c13L5 to be https://standardjs.com/